mailpoet/assets/js/src/wizard/welcome_wizard_controller.jsx

```diff
@@ -101,7 +102,9 @@ function WelcomeWizardStepsController(props) {
 
   return (
     <>
-      <Steps count={stepsCount} current={step} />
+      <TopBar>
```

Comment:
Here I opted to use the TopBar component to be consistent with other parts of the MailPoet admin. The logo using this component has 22px instead of 24px. Also, the TopBar component adds a link to the logo which we don't need here. If we settle on using this component, which I'm inclined to think is the best approach, my plan is to change it by adding a new parameter to make it possible to display the logo without a link.

Reply:
I like this option 👍

Reply:
@shivanarrthine, are you also ok with this change in the logo size to reuse a React component that we are already using in other parts of the admin interface?

Reply:
@rodrigoprimo Yes, that sounds good. Let's reuse the same components.

Reply:
Thanks for your input, @shivanarrthine!